scenes/demos/mobject_intersection/formula_modifications.py

```diff
@@ -9,19 +9,17 @@ def construct(self):
             'a', '$\cdot$', '2', '$\cdot$', 'a', '$\cdot$', 'b',
             font_size=70
         ) # 3aba2a
-        # tex_2.shift([0, -1, 0])
-
 
         self.add(tex_2)
         self.wait()
 
-        self.rearrange_formula(tex_2, [8, 9, 0, 1, 2, 3, 4, 5, 6, 7, 10, 11, 12, 13], [8, 9], []) # 23abaab
+        self.rearrange_formula(tex_2, [8, 9, 0, 1, 2, 3, 4, 5, 6, 7, 10, 11, 12, 13], [8], [], [9], [1]) # 23abaab
```

Comment:
Are you fading out/in the multiplication sign?

Reply:
հա, դոքսթրինգերը չեմ գրել, ֆեյդաութ անելունը սկզբնական բանաձևի համարներն են, իսկ ֆեյդինը՝ վերջնական